src/entities/Place/model.test.ts

```diff
@@ -647,7 +654,7 @@ describe(`findWorld`, () => {
           AND hidden is false
           AND world_name IN ($4)
           AND rank > 0
-        ORDER BY rank DESC, p.like_rate DESC
```

Comment:
If we don't remove like_rate, please test both cases

Reply:
same as prev comment